src/test/java/com/openmrs/migrator/core/services/CommandServiceTest.java

```diff
@@ -0,0 +1,30 @@
+package com.openmrs.migrator.core.services;
+
+import com.openmrs.migrator.core.exception.CommandExecutionException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class CommandServiceTest {
+
+  @Autowired private CommandService commandService;
+
+  @Test
+  public void shouldExecuteLsCommand() {
+    try {
+      commandService.runCommand("ls", "-ltr");
```

Comment:
Is it possible to hide the result from this command? When I run the tests it displays the list of directories in the current folder...

Reply:
Done